### DISCUSSION TITLE
Newer Renogy Modbus Protocol V1.7 Document

### DISCUSSION THREAD

**Discussion by TimGFoley** [Louisvdw/dbus-serialbattery#399] [General]
Found this recently, it covers more of the Smart Battery registers including the heaters etc.

I'll work on updating the Renogy support with some of this new info.

[Lithium.Iron.Battery.BMS.Modbus.Protocol.V1.7.zh-CN.en.1.pdf](https://github.com/Louisvdw/dbus-serialbattery/files/10367319/Lithium.Iron.Battery.BMS.Modbus.Protocol.V1.7.zh-CN.en.1.pdf)

**Comment by adamhratcliffe** [comment]
Hi, not sure if this is the right place to reach out.  My system seems pretty similar to yours in that I'm using Renogy Lithium batteries.  My temperature values do not seem to read correctly.  They are just sat at 25 degrees no matter what. From your images, yours seem to vary and sit lower.
